### PR TITLE
Add sitemap and gdpr plugin, and add plugin tasks

### PR DESCRIPTION
**Pull request by delyriand**

### CODE REVIEW

File: castor.php [hunk 1]
```diff
@@ -111,6 +111,7 @@ function cleanUp(): void
     if (io()->confirm('Are you sure? This is a destructive action!', false)) {
         run('rm -rf castor');
         run('rm castor.php');
+        run('rm -rf themes-examples');
```
Comment (jacquesbh) [inline castor.php:114]: I'm not a huge fan of this.

We don't propose a way to install the custom theme automatically, so removing it is kind of sad.

File: castor/sylius.php [hunk 2]
```diff
@@ -72,6 +90,14 @@ function installPlugins(): void
             while (!io()->confirm('Have you updated your cron.json?', false));
             run('symfony console doctrine:migrations:migrate -n', path: 'apps/sylius'); // Run plugin migrations
         },
+        'stefandoorn/sitemap-plugin' => function () {
+            io()->info('Follow the installation guide: https://github.com/stefandoorn/sitemap-plugin#installation');
```
Comment (jacquesbh) [inline castor/sylius.php:94]: A while loop is missing here to be sure that the developer has done the installation correcly.

File: castor/sylius.php [hunk 3]
```diff
@@ -72,6 +90,14 @@ function installPlugins(): void
...
+            io()->info('Update your clevercloud/cron.json by adding this new line:');
+            io()->block('"0 2 * * *      $ROOT/clevercloud/symfony_console.sh sylius:sitemap:generate",');
+        },
+        'synolia/sylius-gdpr-plugin' => function () {
+            io()->info('Follow the installation guide: https://github.com/synolia/SyliusGDPRPlugin#installation');
```
Comment (jacquesbh) [inline castor/sylius.php:99]: Same here, maybe we need a loop to block the user on going forward before it's really installed?

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: castor.php (hunk 1 above)
Comment (delyriand) [reply castor.php:114]: if you want ;)

**Review by jacquesbh — APPROVED** [review]

**Pull request merged**